### PR TITLE
create webber.yml

### PR DESCRIPTION
**Pull request by dragon540**

### CODE REVIEW

**Comment by dragon540** [comment]
@ritwik12 travis ci build is failing again and again. please check it.

**Comment by ritwik12** [comment]
@dragon540 Because every line is having extra "+". Please correct that.

File: _data/projects/Webber.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Webber
+desc: a WYSIWYG editor for html
+site: https://github.com/dragon540/Webber
+tags:
+- qt
+- cplusplus
+- wysiwyg
+upforgrabs:
+  name: help needed
+  link: https://github.com/dragon540/Webber/issues
```
Comment (shiftkey) [inline _data/projects/Webber.yml]: I think this should be pointed to your `help wanted` label: `https://github.com/dragon540/Webber/labels/help%20wanted`

File: _data/projects/Webber.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: Webber
+desc: a WYSIWYG editor for html
+site: https://github.com/dragon540/Webber
+tags:
+- qt
+- cplusplus
+- wysiwyg
+upforgrabs:
+  name: help needed
```
Comment (shiftkey) [inline _data/projects/Webber.yml]: This should be the label you assign to issues: `help wanted`

**Review by shiftkey — CHANGES_REQUESTED** [review]

**Comment by dragon540** [comment]
Done that.

File: _data/projects/Webber.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/Webber.yml]: @dragon540 You should change the link as mentioned by @shiftkey

**Comment by dragon540** [comment]
@ritwik12 please merge my PR.

File: _data/projects/Webber.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/Webber.yml]: @dragon540 please make this change so that I could merge.

**Comment by ritwik12** [comment]
@dragon540 Please change label link as mentioned by shiftkey

**Comment by dragon540** [comment]
@ritwik12 now please merge my PR.

**Review by shiftkey — APPROVED** [review]

**Comment by shiftkey** [comment]
@dragon540 please be aware that this is something we do in spare time

**Comment by dragon540** [comment]
Sorry for multiple requests.i understand

**Pull request merged**